avocado/utils/script.py

```diff
@@ -39,7 +39,7 @@ class Script(object):
     Class that represents a script.
     """
 
-    def __init__(self, path, content, mode=DEFAULT_MODE):
+    def __init__(self, path, content, mode=DEFAULT_MODE, open_mode='w'):
```

Comment:
It'd make sense to also add `open_mode` to `make_script` and `make_temp_script`, but it can be done later.